code/game/atoms.dm

```diff
@@ -1392,6 +1394,24 @@
 
 		SEND_SIGNAL(src, COMSIG_ATOM_VV_MODIFY_TRANSFORM)
 
+	if(href_list[VV_HK_SPIN_ANIMATION] && check_rights(R_VAREDIT))
+		var/num_spins = input(usr, "How many spins?", "Spin Animation") as null|num
```

Comment:
this should come with a "do you want infinite spins" yes/no prompt

Reply:
-1 is infinite but I guess it isn't immediately clear. Will adjust shortly